packages/reactive-element/src/reactive-element.ts

```diff
@@ -817,6 +821,9 @@ export abstract class ReactiveElement
   }
 
   /**
+   * Install a [[`ReactiveController`]] onto the element's reactive update
```

Comment:
Maybe we should start using e.g. `{@link ReactiveController}`, since that will work in VSCode too. We will need to change or update this regexp to extract them for our lit.dev API docs: https://github.com/lit/lit.dev/blob/e2d7fd3625fb021c539348d7ad06121e871a8ebd/packages/lit-dev-tools/src/generate-api-docs-data.ts#L730

Reply:
(Optional, maybe we should do this all at once with a search/replace).

Reply:
Great point. Will remove links from this change and tracking: https://github.com/lit/lit.dev/issues/503